Documentation/Testing/ProjectTesting.rst

```diff
@@ -100,7 +100,7 @@ TYPO3 testing framework:
 
     composer req --dev typo3/coding-standards typo3/testing-framework
 
-.. _testing-projects-configuration:
+.. _testing-project-configuration:
```

Comment:
Maybe it would be good to keep the old link. No need to create redirects or whatever:
```suggestion
.. _testing-projects-configuration:
```